contentcuration/contentcuration/utils/sync.py

```diff
@@ -234,7 +233,7 @@ def sync_node_assessment_items(node, original):  # noqa C901
         node.changed = True
 
     if ai_to_update:
-        bulk_update(ai_to_update, update_fields=assessment_item_fields)
+        AssessmentItem.objects.bulk_update(ai_to_update, ['assessment_item_fields'])
```

Comment:
This doesn't look correct to me. The second parameter sent to `bulk_update` should be the list of fields to update, but `['assessment_item_fields']` is being passed? In that case `assessment_item_fields` should be a field of `AssessmentItem`, which it is not.

Reply:
@bjester I noticed my mistake. Thank you, I have updated the PR. Could you take a look at it?